docs/setup_vscode_wsl.md

```diff
@@ -0,0 +1,575 @@
+---
+layout: spec
+title: Setup VSCode
+sitemapOrder: 20
+---
+
+Setup up VS Code for C/C++ on Windows
+=====================================
+{: .primer-spec-toc-ignore }
+
+[Visual Studio Code](https://code.visualstudio.com/) is a lightweight, easy-to-use, source code editor with debugging support.  It runs on macOS, Windows, and Linux (including CAEN Linux).  Visual Studio Code is not the same program as Visual Studio.
+
+This tutorial is specific to Windows.  Looking for the [macOS version](setup_vscode_macos.html)?
+
+
+<div class="primer-spec-callout info" markdown="1">
+If you already have VS Code installed with the C/C++ extensions, skip to the [Create a project](#create-a-project) section.
+</div>
+
+## Prerequisites
+VS Code relies on external command line tools.  To install CLI tools, follow the [Windows command line tools tutorial](setup_wsl.html).
+
+Make sure you have a compiler and a debugger installed.  Your version might be different.  Instructions for installation on [Windows/WSL/Linux](setup_wsl.html#install-cli-tools).
+```console
+$ g++ --version
+g++ (GCC) 8.5.0 20210514
+$ gdb --version
+GNU gdb (GDB)
+```
+
+Next, follow our [Command line interface (CLI)](cli.html) tutorial.
+
+<div class="primer-spec-callout warning" markdown="1">
+**Pitfall:** Make sure you have installed [CLI tools for Windows](setup_wsl.html#install-cli-tools) before continuing.
+</div>
+
+## Install
+Make sure you have updated Windows and WSL installations according to the [WSL tutorial](setup_wsl.html).
+
+Install VS Code from the web [https://code.visualstudio.com/](https://code.visualstudio.com/).
+
+Select "Add to PATH".
+
+<img src="images/vscode005.png" width="480px" />
+
+Reboot.  Open the Terminal ([Ubuntu Bash Shell](cli.html#open-terminal-windows)) and verify your installation.  Your version might be different.
+
+```console
+$ code --version
+1.74.1
+1ad8d514439d5077d2b0b7ee64d2ce82a9308e5a
+x64
+```
+
+### Extensions
+Make sure VS Code is installed correctly by checking the version.  You need version 1.52.1 or higher.
```

Comment:
This feels redundant since it immediately follows the previous section where we have students do the same thing. I suggest consolidating them. This would also help make it more clear that the Ubuntu/bash terminal (only mentioned in the previous section) is what they need to use for installing extensions (i.e. not PowerShell).

Reply:
Fixed 28f2d0ae66ebac0baada4b42b27f360cd847bc48